uui/components/navigation/MainMenu/MainMenu.tsx

```diff
@@ -9,7 +9,7 @@ export interface MainMenuMods {}
 function applyMainMenuMods() {
     return [
         css.root,
-        'uui-mainMenu',
+        'uui-main_menu',
```

Comment:
let's avoid this change to not cause a breaking change

Reply:
this is not a breaking change, it is implemented within the framework of this RC, I just tweaked the class, it's more correct than using a camelcase